### ISSUE TITLE
Review hosted documentation options

### ISSUE THREAD

**Issue by davidjb** [jcu/cookbook#35]
The documentation for this project needs a cleanup and a simplification.  At present, https://web.jcu.io is the old v1 documentation of the JCU Web Framework (still currently hosted at https://github.com/jcu/cookbook/tree/gh-pages) and this needs to be archived off to an "old" URL (such as https://cookbook.jcu.io/docs/v1) and should be replaced with a new, simpler set of documentation about CookBook (either the current v2 or the proposed v3, related to #34).

The ideal scenario is a situation where we can have a base set of Bootstrap documentation that we can add to, theme accordingly and then publish.  At present, CookBook v2 has an attempt at doing this by way of a single-page HTML document (https://github.com/jcu/cookbook/blob/master/site/docs/2.0/index.html) that is based upon Bootswatch's documentation (https://bootswatch.com/default/).  This documentation isn't currently being served on a URL directly and requires heavy modification - but that latter part may not be something we can optimise for.  Likewise, v1 of the Web Framework attempted to build upon the official Bootstrap documentation (e.g. https://github.com/twbs/bootstrap/), but as the docs were/are embedded in the Bootstrap source itself, this required maintaining essentially a fork of Bootstrap - not a simple feat.

Edit: turns out Bootstrap has a cheat sheet built in - https://getbootstrap.com/docs/5.1/examples/cheatsheet/ (it's currently broken but will be [fixed soon](https://github.com/twbs/bootstrap/issues/34806); using this 'official' source of documentation would be better than Bootswatch.

In short, we should review what bases these are for building documentation (HTML or other documentation projects, NPM, etc) for Bootstrap to see if any fit our requirements.  From there, we need processes and/or scripts for adapting and adding to the documentation, as well as publishing from here on out.

**Comment by davidjb** [comment]
Additionally, being able to document (and automatically update) usage snippets for the CSS/JS to feature [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) hashes would be ideal in the same way that https://getbootstrap.com/docs/5.0/getting-started/download/ does.

**Comment by davidjb** [comment]
This has been resolved in #37 and SRI hashes implemented in 09ff4b607f6754473ee0e76f36106de461d0fb12.  Awesome work @TimEnglart!
